pkg/ivd/ivd_protected_entity_type_manager.go

```diff
@@ -40,36 +41,47 @@ type IVDProtectedEntityTypeManager struct {
 	vsom      *vslm.GlobalObjectManager
 	cnsClient *cns.Client
 	s3URLBase string
-	user      string // These are being kept so we can open VDDK connections, may be able to open a VDDK connection
-	password  string // in IVDProtectedEntityTypeManager instead
+	vcParams  map[string]interface{} // Save the VC configuration params
 	logger    logrus.FieldLogger
 }
 
 func NewIVDProtectedEntityTypeManagerFromConfig(params map[string]interface{}, s3URLBase string,
 	logger logrus.FieldLogger) (*IVDProtectedEntityTypeManager, error) {
+	logger.Infof("Creating NewIVDProtectedEntityTypeManagerFromConfig. Params: %v", params)
```

Comment:
Would this log the password?, I think logging the username is sufficient.

Reply:
echo it. We should not log any password.

Reply:
Good catch. Maybe we should not print any params from the secret.